nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/state/shared/index.ts

```diff
@@ -451,6 +451,8 @@ export interface ComponentReference {
     id: string;
     parentGroupId?: string;
     name: string;
+    targetUri?: string;
+    flowRefreshed?: string;
```

Comment:
Rather than updating this shared type, can we explicitly store the needed details for the RPG in the mange remote ports store?